templates/header.war.html

```diff
@@ -13,64 +13,10 @@
     e.g. <a href="https://adoptium.net/">Eclipse Temurin</a>.
   </p>
 
-  <h2>
-    Weekly Release Line
-  </h2>
-
   <p>
-    Supported Java versions for the weekly release line are:
+    To determine the Java version that is supported for your Jenkins environment, please refer to the <a href="https://www.jenkins.io/doc/book/platform-information/support-policy-java/#running-jenkins-system">Java support policy.</a>
   </p>
 
-  <dl>
-    <dt>2.463 (June 2024) and newer</dt>
-    <dd>Java 17 or Java 21</dd>
-
-    <dt>2.419 (August 2023) and newer</dt>
-    <dd>Java 11, Java 17, or Java 21</dd>
-
-    <dt>2.357 (June 2022) and newer</dt>
-    <dd>Java 11 or Java 17</dd>
-
-    <dt>2.164 (February 2019) and newer</dt>
-    <dd>Java 8 or Java 11</dd>
-
-    <dt>2.54 (April 2017) and newer</dt>
-    <dd>Java 8</dd>
-
-    <dt>1.612 (May 2015) and newer</dt>
-    <dd>Java 7</dd>
```

Comment:
The Java 8 and Java 7 items are not included in the current [Java support policy page](https://www.jenkins.io/doc/book/platform-information/support-policy-java/).  Before we merge this, I think that we should include them on that page.

Reply:
Thanks for the reminder @MarkEWaite, I've submitted the PR for that https://github.com/jenkins-infra/jenkins.io/pull/7726 and have updated the text to use "on" instead of "through"